ios-release-build.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: page
+title: Preparing an iOS App for Release
+sidebar: home_sidebar
+permalink: /ios-release/
+---
+
+During development, you'll test an app using `flutter run` at the command line,
+using the Run and Debug toolbar buttons in IntelliJ, or both. By default,
+Flutter builds a *debug* version of your app.
```

Comment:
I wonder if this paragraph adds anything? Maybe we should just kill it and go directly to the point?